lib-ce/emqx_modules/src/emqx_mod_delayed.erl

```diff
@@ -58,6 +59,7 @@
 -define(TAB, ?MODULE).
 -define(SERVER, ?MODULE).
 -define(MAX_INTERVAL, 4294967).
+-define(ACL_PRIORITY, 10000).
```

Comment:
we need one single place to manage the priorities.

maybe a new API `emqx_hooks:priority/1`

`priority(pre_ack_checks) -> 10000.`

in the future, we may add more conventions.

Ping @savonarola for comments.

Reply:
OK. we can do it in the future.
 Resolving the comment now.